CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [3.0.0](https://github.com/react-native-community/fetch/compare/v2.0.0...v3.0.0) (2021-08-02)
+
+### Bug Fixes
+
+* do not return a promise from BlobResponseConstructor ([3745808](https://github.com/react-native-community/fetch/pull/12/commits/37458086f22e04b11e7fb2bb5e701999f1def02e))
+
```

Comment:
Please revert this change. The changelog is updated automatically during the release process.

Reply:
aha